charades-server/README.md

```diff
@@ -0,0 +1,27 @@
+# Installation
+
+Remember to use virtual environment.
+
+### Set virtual environment
+
+```python
+py -m venv venv_charades
```

Comment:
Let's add instruction for Unix systems
`py` works only on Windows, in unix you have to use the `python3` command (or create alias)

Reply:
I think that worked on my WSL, but sure